fl4health/strategies/feddg_ga.py

```diff
@@ -23,7 +30,7 @@ class FairnessMetricType(Enum):
     """Defines the basic types for fairness metrics, their default names and their default signals"""
 
     ACCURACY = "val - prediction - accuracy"
-    LOSS = "val - loss"
```

Comment:
Changing to checkpoint, as this is the name of the vanilla loss for basic clients during validation.